app/assets/stylesheets/application.scss

```diff
@@ -13,6 +13,7 @@ $govuk-page-width: 1140px;
 
 @import "./admin/overrides";
 @import "./admin/layout";
+@import "./admin/summary";
```

Comment:
All the classes begin with `app-view-summary` so i think this file should probably be 
```
@import "./admin/view/ssummary";
```